compiler/src/iree/compiler/Codegen/LLVMGPU/Utils/LLVMGPUSelectUKernels.cpp

```diff
@@ -29,13 +30,35 @@ getUKernelNameAndSuffixForArgmax(linalg::GenericOp op) {
                                   indexType.getElementType())};
 }
 
+// Returns ukernel name and suffix for multi_mma. Empty name = no ukernel.
+static std::tuple<std::string, std::string>
```

Comment:
Now that we have 3 functions with the name and suffix combo, consider defining a struct with named fields for these?